echopype/echodata/echodata.py

```diff
@@ -67,27 +67,40 @@ def __init__(
         self.converted_raw_path: Optional["PathHint"] = converted_raw_path
         self._tree: Optional["DataTree"] = None
 
-        # object associated with directly writing to a zarr file
-        self.parsed2zarr_obj = parsed2zarr_obj
-
         self.__setup_groups()
         # self.__read_converted(converted_raw_path)
 
         self._varattrs = sonarnetcdf_1.yaml_dict["variable_and_varattributes"]
 
-    def cleanup(self):
-        if (self.parsed2zarr_obj is not None) and (self.parsed2zarr_obj.store is not None):
-            # get Path object of temporary zarr file created by Parsed2Zarr
-            p2z_temp_file = self.parsed2zarr_obj.store
-
-            # remove temporary directory created by Parsed2Zarr, if it exists
-            if p2z_temp_file.fs.exists(p2z_temp_file.root):
-                p2z_temp_file.fs.rm(p2z_temp_file.root, recursive=True)
+    def cleanup_swap_files(self):
+        """
+        Clean up the swap files during raw data conversion
+        """
+        sonar_group = "Sonar"
+        beam_group_var = "beam_group"
+        for beam_group in self[sonar_group][beam_group_var].to_numpy():
```

Comment:
This is just a question: why is `.to_numpy` needed here?

Reply:
This is getting the `beam_group` labels such as `Beam_group1`, `Beam_group2`, etc, so the values are needed here.